examples/from-conf/main.go

```diff
@@ -28,12 +28,11 @@ func main() {
 		log.Fatal(err)
 	}
 	err = sender.
-		Table("trades_go").
```

Comment:
FYI @insmac I originally added a suffix to each table with the name of each client language, in case we decided to validate the ingested data in the client test pipeline. But I don't think we're ever going to do this, so I think that using the same table across languages in that pipeline is fine.